pkg/cmd/task/list.go

```diff
@@ -0,0 +1,138 @@
+// Copyright © 2019 The Tekton Authors.
```

Comment:
should we rename all files as well in a separate PR?

Reply:
@sthaha so far I am just adding this to new file — but in `tektoncd/cli` it applies to all, so yeah :sunglasses: